website/api/zkvm/tutorials/io.md

```diff
@@ -0,0 +1,329 @@
+# Understanding I/O in the zkVM
+
+In the [Hello World Tutorial][docs-hello-world], we had a brief introduction to
+how to perform I/O operations in the zkVM. Now we'll dive deeper into the subject.
+By following the steps in this guide, you will learn:
```

Comment:
This phrase sets the reader up to expect a step-by-step tutorial or how-to guide, when this is actually an [explanation(https://documentation.divio.com/explanation.html). Suggesting a specific revision, but mostly I just want to remove the word "steps." 

```suggestion
Keep reading to learn more about:
```

Reply:
(or if we want to keep the word "steps," let's revise the text to be clear about in what sense this is a step-by-step guide.

Reply:
Removing "steps" seems good to me, thanks for the feeback!